docs/basics/functions.md

```diff
@@ -832,13 +832,14 @@ Which would return the following:
 }]
 ```
 
-### split_array
+### split_array (leaf_array)
 
-| Applies to | Description                           |
-| :--------- | :------------------------------------ |
-| API        | Split an array that has nested arrays |
+| Applies to | Description |
+| :--------- | :---------- |
+| API | Split an array that has nested arrays |
+||With leaf_array option, turn array leaf node(s) into sample(s)
```

Comment:
Enable the `leaf_array`option to turn array leaf nodes into samples

Reply:
Updated functions.md accordingly.